msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -178,7 +178,18 @@ For more information on the features available see: [Features in the Public Deve
      "contoso.com",
      "mysite.someplace.com",
      "othersite.someplace.com"
-  ]
+  ],
+   "configurableProperties": [
+     "name",
+     "shortDescription",
+     "longDescription",
+     "smallImageUrl", // The `smallImageUrl` is the `outline` property in the `icons` block of the manifest.
```

Comment:
Could you please remove // comments which are in front of smallImageUrl & largeImageUrl? This makes JSON invalid.